### ISSUE TITLE
Bug when rending multiple lines of lyrics.

### ISSUE THREAD

**Issue by JonathanILevi** [paulrosen/abcjs#154]
When you render multiple lines of text the renderer creates to big gaps between the words.

It looks like it calculates the minimum width for the word column with the total length of every word and not just the widest word.

Render this and it will show the problem.

X:1
K:Dmaj
[V:s0] A | F/ G/Ad | c/ B/A || B | A/ G/FE/D/ | E2 ||
w:Word word word word word word word word word word word word word word word 
w:word word word word word word wo
w:word word word wo
w:word
w:word
w:word
w:word
w:word

**Comment by JonathanILevi** [comment]
It is fixed, thanks!

**Comment by JonathanILevi** [comment]
Um, you may have created another bug in the process of fixing this one.

Render this:
X: 1
K: Emin
|:A A A A A
w:a a * a a
w:a a a a a

In abcjs 5.1.2 the middle "a" in the second line is half a line to high; in abcjs 5.0.0 it is rendered at the correct height.

Thanks again,

**Comment by paulrosen** [comment]
Ok, that will be fixed in v5.1.3.

**Comment by paulrosen** [comment]
Well, I haven't been able to reproduce this. Do you have this happening on a public facing site? For instance, if you go here: https://editor.drawthedots.com/ and paste in that tune, it looks ok.

**Comment by JonathanILevi** [comment]
I believe "https://editor.drawthedots.com/" still uses an earlier version.

Doing this will show the error.

If you go to this site "https://abcjs.net/abcjs-editor.html" and render the given abc it will render the "a" correctly.  (This site uses version 5.0.0)

Copy and paste the abcjs 5.1.2 data into the console (from here "https://raw.githubusercontent.com/paulrosen/abcjs/master/bin/abcjs_midi_5.1.2-min.js")

Then in order to render with the new version run this in the console `ABCJS.renderAbc("paper0","X: 1\nK: Emin\n|:A A A A A\nw:a a * a a\nw:a a a a a")`

BTW I am using Firefox on Linux Mint.

**Comment by paulrosen** [comment]
I updated https://abcjs.net/abcjs-editor.html to the latest code - thanks for reminding me. If I go to the page now and paste in your example, it still looks right to me on chrome, firefox, and safari. What device and browser are you using?

**Comment by JonathanILevi** [comment]
Okay, I figured out the problem.

Somehow the abcjs source on my system got converted to ascii.
I think I caused it by rather then properly downloading the abcjs-min file I copied the entire file contents as the browser showed it to a newly created file in my system.  I looks like any unicode characters in the source got converted to ASCII because of how the browser displayed the file.

Thank for your help.
